src/script/common/c_content.cpp

```diff
@@ -1209,7 +1209,7 @@ void push_inventory_list(lua_State *L, Inventory *inv, const char *name)
 	}
 	std::vector<ItemStack> items;
 	for(u32 i=0; i<invlist->getSize(); i++)
-		items.push_back(invlist->getItem(i));
+		items.push_back((ItemStack) invlist->getItem(i));
```

Comment:
The function returns either an ItemStack const or an ItemStack. 
It was for my IDE to be happy. I remove, but we could cast in (const ItemStack) right?